src/main/java/se/sundsvall/educationfinder/service/StatisticsService.java

```diff
@@ -38,13 +38,13 @@ public StatisticsService(final CourseRepository courseRepository, final SubjectR
 		this.subjectRepository = subjectRepository;
 	}
 
-	public Statistics getStatisticsByParameters(final StatisticsParameters parameters) {
-		var codes = getSubjectCodes(parameters);
-		var courses = courseRepository.findAllByParametersAndCode(parameters, codes);
+	public Statistics getStatisticsByParameters(final StatisticsParameters parameters, final String municipalityId) {
```

Comment:
Maybe skip adding municipalityId-parameter to service method?

We have it in the API now. When (if) we need it in the service class, it will be an easy task to add it.
(this will also remove the accepted sonar issue in this PR)

Comment applicable for both service-classes in this PR.